client_app/src/components/shelter/RequestForHelp.js

```diff
@@ -0,0 +1,37 @@
+import { Calendar, dayjsLocalizer } from "react-big-calendar";
+import dayjs from "dayjs";
+import React, { Component } from "react";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+
+const localizer = dayjsLocalizer(dayjs);
+
+class RequestForHelp extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      events: [
+        {
+          start: dayjs().toDate(),
+          end: dayjs().add(1, "day").toDate(),
+          title: "Some title",
+        },
+      ],
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <Calendar
+          localizer={localizer}
+          defaultDate={new Date()}
+          defaultView="month"
+          events={this.state.events}
```

Comment:
Should we also implement some sort of drop down from the months section (able to select a month from a drop down box and not have to click the arrows for easier access)